spring-integration-cassandra/src/test/java/org/springframework/integration/cassandra/config/CassandraOutboundAdapterParserTests.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.cassandra.config;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+import org.springframework.beans.factory.parsing.BeanDefinitionParsingException;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+import org.springframework.integration.cassandra.outbound.CassandraMessageHandler;
+import org.springframework.integration.test.util.TestUtils;
+
+
+/**
+ * @author Filippo Balicchia
+ *
+ */
+public class CassandraOutboundAdapterParserTests {
+	@Test
+	public void minimalConfig() {
+		ClassPathXmlApplicationContext context = new ClassPathXmlApplicationContext("outbound-adapter-parser.xml",
```

Comment:
Would be better to make this test class based on the `@RunWith(SpringJUnit4ClassRunner.class)` anyway: less context loading - better performance!
Yes, the `outbound-adapter-parser.xml` should be renamed to the `CassandraOutboundAdapterParserTests-context.xml` to be loaded by the Spring Test Framework.